.github/dependabot.yml

```diff
@@ -1,4 +1,10 @@
 version: 2
+registries:
+  ghcr:
+    type: docker-registry
+    url: ghcr.io
+    username: x
+    password: ${{ secrets.BASE_CONTAINER_IMAGE_READER_DEPENDABOT }}
```

Comment:
I moved this to the top because that's where we typically have been putting the `registries` key on all our files for easier skimmability...